addons/binding/org.openhab.binding.konnected/src/main/java/org/openhab/binding/konnected/internal/handler/KonnectedHandler.java

```diff
@@ -82,19 +81,22 @@ public KonnectedHandler(Thing thing, String path, String hostAddress, String por
     public void handleCommand(ChannelUID channelUID, Command command) {
         // get the zone number in integer form
         Channel channel = this.getThing().getChannel(channelUID.getId());
+        String channelType = channel.getChannelTypeUID().getAsString();
         String zoneNumber = (String) channel.getConfiguration().get(CHANNEL_ZONE);
         Integer zone = Integer.parseInt(zoneNumber);
         logger.debug("The channelUID is: {} and the zone is :", channelUID.getAsString(), zone);
         // convert the zone to the pin based on value at index of zone
         Integer pin = Arrays.asList(PIN_TO_ZONE).get(zone);
         // if the command is OnOfftype
         if (command instanceof OnOffType) {
-            int sendCommand = (OnOffType.OFF.compareTo((OnOffType) command));
-            logger.debug("The command being sent to pin {} for channel:  is {}", channelUID.getAsString(), pin,
-                    sendCommand);
-            sendActuatorCommand(Integer.toString(sendCommand), pin, channelUID);
-        } else if (command instanceof OpenClosedType) {
-            logger.debug("A command was sent to a sensor type so we are ignoring the command");
+            if (channelType.equalsIgnoreCase(CHANNEL_SWITCH)) {
+                logger.debug("A command was sent to a sensor type so we are ignoring the command");
+            } else {
+                int sendCommand = (OnOffType.OFF.compareTo((OnOffType) command));
+                logger.debug("The command being sent to pin {} for channel:  is {}", channelUID.getAsString(), pin,
```

Comment:
You have 3 parameters, but only two `{}`.

Reply:
fixed

Reply:
added the correct parameters in the appropriate order